localgovdrupal/localgov_alert_banner
# Feature: Replace js cookie

## What does this change?

Replaces js-cookie functionality with native Javascript.
Resolves #352 

## How to test

- Create a mix of alert banners and full page alert banners.
- Open the site homepage.
- Hide one or more banners.
- Reload the page.
- The hidden banners should remain hidden.
